internal/policy/src/main/resources/policy/checkstyle.xml

```diff
@@ -16,23 +16,25 @@
 <module name="Checker">
     <property name="fileExtensions" value="java,properties,xml"/>
 
-    <module name="SuppressWarningsFilter" />
+    <module name="SuppressWarningsFilter"/>
+
+    <module name="FileTabCharacter"/>
 
     <module name="NewlineAtEndOfFile">
         <property name="lineSeparator" value="lf"/>
     </module>
 
     <module name="RegexpMultiline">
         <property name="format" value="(?s:(\r\n|\r).*)"/>
-        <property name="message" value="Line has CRLF or CR line ending."/>
+        <property name="message" value="Line has CRLF or CR line ending"/>
```

Comment:
not sure about the rewording changes. Why are we removing the '.' ?

Reply:
I wanted it to be consistent. The style was mixed, some Checkstyle messages with trailing period, some without.